source/statements.tex

```diff
@@ -815,7 +815,8 @@
 in a function whose return type is not \cv{}~\keyword{void};
 \indextext{conversion!return type}%
 the \tcode{return} statement initializes the
-glvalue result or prvalue result object of the (explicit or implicit) function call
+returned reference or prvalue result object
```

Comment:
If we prepare to consistently use "Y is X-initialization from Z" to specify which initialization rule will apply to that construct, in the standard. As indicated in the #4582 issue. For this purpose, I think it would be adequately clear by using this wording without using an invented declaration.

Reply:
The issue was that initialization of a "glvalue result" is nowhere defined, but initialization of a reference is. We also say it's copy-initialization, so I think we're good, at least for this particular situation.